synthtool/gcp/templates/node_split_library/index.ts.j2

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by synthtool. **
+// ** https://github.com/googleapis/synthtool **
+// ** All changes to this file may be overwritten. **
+
+{% for version in versions %}import * as {{ version }} from './{{ version}}';{{ "\n" }}{% endfor %}
```

Comment:
Out of curiosity, why are we putting these new templates in their own templates directory, did this make it easier to include the additional templates conditionally?

I notice that upstream you still call:

```python
templates = common_templates.node_library(
    source_location='build/src', versions=versions, default_version="v1")
```

Reply:
right. synthtool will copy all files from `gcp/templates/node_library`, I could put this template in `node_library` and add some logic to exclude this file. This make it easier to include additional templates.